DataManagement/GitTutorial.Rmd

```diff
@@ -92,76 +95,65 @@ In RStudio click on **Tools > Global Options**, select **Git/SVN** tab.
 Ensure that the path to the Git executable is correct.
 This is particularly important in Windows where it may not default correctly (e.g. C:/Program Files (x86)/Git/bin/git.exe).
 
-Click **Create RSA Key**
-<br/><br/>
-```{r git-svn}
-knitr::include_graphics("Pics/gittutorial/2_git_svn.png")
-```
-<br/><br/>
-<br/><br/>
+GitHub needs to validate who you are before you can connect it and RStudio.
+We can do this by generating and saving a Personal Access Token (PAT).
+You only need to do this once.
 
-Click **Create** and close this window.
-<br/><br/>
-```{r create-key}
-knitr::include_graphics("Pics/gittutorial/3_create_key.png")
+```{r PAT, eval = FALSE, echo = TRUE}
+create_github_token()
 ```
-<br/><br/>
-<br/><br/>
+This function will open GitHub. 
+After confirming your password, you will be shown a page to make a new PAT.
+You don't need to change any of the options.
+Just click the green "Generate token" button at the bottom of the page.
 
-Back on the previous window click **View public key**. And copy the displayed **Public Key**.
-<br/><br/>
-```{r public-key}
-knitr::include_graphics("Pics/gittutorial/4_public_key.png")
-```
-<br/><br/>
-<br/><br/>
-Go to your GitHub account, click in the upper right corner and open your account **Settings**.
-<br/><br/>
-```{r settings}
-knitr::include_graphics("Pics/gittutorial/5_settings.png")
-```
-<br/><br/>
-<br/><br/>
+The next page will show you your PAT. 
+Copy it by clicking on the clipboard icon and return to R.
 
```

Comment:
Say something that you will need the PAT later to save it somewhere safe, so don't loose it. Or go straight to the next step or something